docs/changehistory/NextVersion.md

```diff
@@ -57,6 +57,36 @@ For the stop sign example described above, you might have a [glTF model](https:/
 
 Context reality models that have been attached using `DisplayStyleState.attachRealityModel`, can now be hidden by turning ON the `ContextRealityModel.invisible` flag.  Previous implementation requiered context reality models to be detached in order to hide it from the scene.
 
+### Contour Display
```

Comment:
Add to table of contents.

Reply:
I will also add "Context Reality model visibility" which is not in the TOC.

Reply:
Updated.